.github/workflows/CodeCoveraReport.yaml

```
@@ -0,0 +1,77 @@
+name: Code coverage report to Codecov
+
+on:
+    push:
+      branches: 
+        - master
```

minor indentation inconsistency
```suggestion
on:
  push:
    branches: 
      - master
```